clang/test/CIR/Lowering/ThroughMLIR/declaration.c

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -triple x86_64-unknown-linux-gnu -fclangir -fno-clangir-direct-lowering -emit-mlir=core %s -o %t.mlir 
+// RUN: FileCheck --input-file=%t.mlir %s
+
+
+int declaration(int x);
+
+// CHECK: func.func private @declaration(i32) -> i32
+
+int declaration_test() {
+    // Variables
+    int number = 15;
+
```

Comment:
Please use 2 spaces, remove this comment (which doesn't add anything) and remove the newlines:
```
int declaration(int x);
int declaration_test() {
  int number = 15;
  return declaration(number);
}
```